edgelet/build/linux/package.sh

```diff
@@ -89,6 +107,16 @@ case "$PACKAGE_OS.$PACKAGE_ARCH" in
         '
         ;;
 
+    redhat8.amd64)
+        SETUP_COMMAND=$'
+            dnf distro-sync -y &&
```

Comment:
Where does `dnf` pull the remote repository from? 
Do I need to setup a different multi-arch linux repository from the CentOs7 for the release artifacts?

Reply:
We should need a different multi-arch repo.  It's a replacement for Yum and can use the same repos.